homie5/values/integer.yml

```diff
@@ -2,25 +2,33 @@ description: validating boolean values
 
 tests:
   - description: Normal integer value without format works
-    type: propertyvalueinteger
+    testtype: propertyvalueinteger
     definition:
       datatype: integer
     input_data: "12"
     output_data: 12
     valid: true
   - description: Rounded step for min max bounds works
-    type: propertyvalueinteger
+    testtype: propertyvalueinteger
     definition:
       datatype: integer
       format: "0:10:2"
     input_data: "9"
     output_data: 10
     valid: true
   - description: Rounded step for max bounds works
-    type: propertyvalueinteger
+    testtype: propertyvalueinteger
     definition:
       datatype: integer
       format: ":10:2"
     input_data: "9"
     output_data: 8
```

Comment:
isn't this the one we wanted to round up?

Reply:
let's take that discussion back to https://github.com/homieiot/convention/issues/301

Reply:
I'll merge this for now.